build.gradle

```diff
@@ -1,8 +1,12 @@
 plugins {
     // Apply the java-library plugin for API and implementation separation.
     id 'java-library'
+    id 'com.jfrog.artifactory' version '4.28.1'
+    id "org.sonarqube" version "3.3"
 }
 
+def projectName = 'Git Files Blame'
```

Comment:
is it still needed as it is specified in gradle.properties?

Reply:
removed in https://github.com/SonarSource/git-files-blame/commit/c8b3a0585d56583c500ca4f3f411fc3a481a13d1